.github/workflows/ci-go.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
       - uses: actions/setup-go@v2
         with:
-          go-version: v1.16
+          go-version: v1.20.x
```

Comment:
Using x instead of fix number will use the latest patch release for Go 1.20

Reply:
> @Aryan-sharma11 Is `x` a valid version number? Should we use a fixed number instead?

Using x instead of fix number will use the latest patch release for Go 1.20

Reply:
I see. Let's hope it doesn't break anytime soon. : )